lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
+        course_key = CourseKey.from_string(course_id)
+    except InvalidKeyError:
+        return render_to_response(invalid_template_path, context)
+    course = modulestore().get_course(course_key)
+    if not course:
+        return render_to_response(invalid_template_path, context)
+
+    # Certificate Lookup
+    try:
+        certificate = GeneratedCertificate.objects.get(
+            user=request.user,
+            course_id=course_key
+        )
+    except GeneratedCertificate.DoesNotExist:
+        return render_to_response(invalid_template_path, context)
+
+    # Load static output values from configuration,
+    configuration = CertificateHtmlViewConfiguration.get_config()
+    context = configuration.get('default', {})
+    # Override the defaults with any mode-specific static values
+    context.update(configuration.get(certificate.mode, {}))
+    # Override further with any course-specific static values
+    context.update(course.cert_html_view_overrides)
+
+    # Populate dynamic output values using the course/certificate data loaded above
+    user_fullname = request.user.profile.name
+    company_name = context.get('company_name')
+    context['accomplishment_copy_name'] = user_fullname
+    context['accomplishment_copy_course_org'] = course.org
+    context['accomplishment_copy_course_name'] = course.display_name
+    context['certificate_id_number'] = certificate.verify_uuid
+    context['certificate_verify_url'] = _("{prefix}{uuid}{suffix}").format(
+        prefix=context.get('certificate_verify_url_prefix'),
+        uuid=certificate.verify_uuid,
+        suffix=context.get('certificate_verify_url_suffix')
+    )
+    context['logo_alt'] = company_name
+
+    accd_course_org_html = '<span class="detail--xuniversity">{org}</span>'.format(org=course.org)
+    accd_company_name_html = '<span class="detail--company">{company_name}</span>'.format(company_name=company_name)
+    # Translators:
+    context['accomplishment_copy_course_description'] = _('a course of study offered by {org}, through {name}.').format(
+        org=accd_course_org_html,
+        name=accd_company_name_html
+    )
+
+    # Translators:
+    context['accomplishment_more_title'] = _("More Information About {user_name}'s Certificate:").format(
+        user_name=user_fullname
+    )
+
+    # Translators:
+    context['certificate_date_issued_title'] = _("Issued On:")
+
+    # Translators:
+    context['certificate_date_issued'] = _('{month} {day}, {year}').format(
+        month=certificate.modified_date.strftime("%B"),
+        day=certificate.modified_date.day,
+        year=certificate.modified_date.year
+    )
+
+    # Translators:
+    context['certificate_id_number_title'] = _('Certificate ID Number')
+
+    # Translators:
+    context['certificate_info_title'] = _('About {company_name} Certificates').format(
+        company_name=company_name
+    )
+
+    cert_info_honor_code_html = "<a href='{tos_url}'>{company_name} Honor Code</a>".format(
```

Comment:
i18n for something like this could look like:
```python
partial = u"<a href='{tos_url}'>" + _("{platform_name} Honor Code") + u"</a>"
cert_info_honor_code_html = partial.format(...)
```

Don't forget to cast the `<a> </a>` strings to unicode because `_()` will return a unicode object